bootstrap/pkg/apis/apps/configconverters/v1beta1.go

```diff
@@ -141,11 +142,8 @@ func (v V1beta1) ToKfConfig(kfdefBytes []byte) (*kfconfig.KfConfig, error) {
 			continue
 		}
 		src := &kfconfig.SecretSource{}
-		if secret.SecretSource.LiteralSource != nil {
-			src.LiteralSource = &kfconfig.LiteralSource{
-				Value: secret.SecretSource.LiteralSource.Value,
-			}
-		} else if secret.SecretSource.EnvSource != nil {
+		// We don't want to store HashSource and LiteralSource explictly.
```

Comment:
nit: Please expand the comment to explain why we don't serialize it (e.g. because we want the config to be checked into source control and don't want secrets in source control).

Can we add a unittest to? e.g. verify that if hashed source or literalsource is set and we call writeconfig the secret gets stripped out.